action/pipeline/exec.go

```diff
@@ -46,7 +46,11 @@ func (c *Config) Exec(client compiler.Engine) error {
 	b.SetBranch(c.Branch)
 	b.SetDeploy(c.Target)
 	b.SetEvent(c.Event)
-	b.SetRef(c.Tag)
+	if c.Tag == "" && c.Event == constants.EventPull {
+		b.SetRef("refs/pull/1")
+	} else {
+		b.SetRef(c.Tag)
+	}
```

Comment:
wouldn't this set the ref to `c.Tag` when `c.Event != constants.EventPull`?

Reply:
Yes. Are you thinking that we should automatically set the ref for other event types if `c.Tag` is not defined?

Reply:
I think we're good. I didn't see that we were setting `c.Tag` previously. Just seems odd that we use `c.Tag` on all but Pull Requests.. sorry for the noise :)